packages/commonwealth/client/scripts/views/components/component_kit/cw_component_showcase.tsx

```diff
@@ -1213,6 +1224,10 @@ export const ComponentShowcase = () => {
           <CWUpvote voteCount={99999} disabled />
         </div>
       </div>
+      <div className="searchbar-gallery">
+        <CWText type="h3">SearchBar</CWText>
+        <CWSearchBar placeholder="Search Common" />
```

Comment:
`Search Common` is a default value for placeholder prop, so you dont need to pass it here

Reply:
Addressed.